recipes/rfc3986/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{%set name = "rfc3986" %}
+{%set version = "0.3.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "b94638db542896ccf89dc62785ec26dbcbd6a97d337f64e02615b164b974f2e5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: https://rfc3986.readthedocs.org
+  license: Apache 2.0
+  summary: 'Validating URI References per RFC 3986'
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
```

Comment:
To save you a few pings, if you want, @sigmavirus24, you could add your maintainer preferences (e.g. some list of packages or stuff you are a developer on) in this issue ( https://github.com/conda-forge/staged-recipes/issues/139 ).